stf/src/cli_utils/account_parsing.rs

```diff
@@ -43,16 +44,29 @@ pub fn get_accountid_from_str(account: &str) -> AccountId {
     }
 }
 
-// get a pair either form keyring (well known keys) or from the store
-pub fn get_pair_from_str(matches: &ArgMatches<'_>, account: &str) -> sr25519::AppPair {
+/// get a pair either form keyring (well known keys) or from the UNTRUSTED keystore
+pub fn get_pair_from_str_untrusted(account: &str) -> sr25519::AppPair {
+    let keystore_path = get_untrusted_keystore_path();
+    get_pair_from_str(keystore_path, account)
+}
+
+/// get a pair either form keyring (well known keys) or from the TRUSTED keystore
+pub fn get_pair_from_str_trusted(matches: &ArgMatches<'_>, account: &str) -> sr25519::AppPair {
+    let keystore_path = get_trusted_keystore_path(matches);
+    get_pair_from_str(keystore_path, account)
+}
```

Comment:
We have a `get_pair_from_str` method for both the trusted and untrusted side. The only difference is the underlying keystore they're using

Reply:
we should someday get rid of this redundancy I think